api-report/container-runtime.api.md

```diff
@@ -553,9 +553,12 @@ export interface ISummaryAckMessage extends ISequencedDocumentMessage {
 
 // @public (undocumented)
 export interface ISummaryBaseConfiguration {
+    compressionAlgorithm?: SummaryCompressionAlgorithm;
```

Comment:
@justus-camp - Any opportunities for reuse / convergence with op compression?  (e.g., configuration patterns to duplicate, sharing an enum of compression algorithms?)

Reply:
The work is only in next for now (need to rebase it onto main), but there's an enum of compression algorithms [here](https://github.com/microsoft/FluidFramework/blob/next/packages/runtime/container-runtime/src/containerRuntime.ts#L479) (currently only lz4). For configuration, I've added `ICompressionRuntimeOptions` [here](https://github.com/microsoft/FluidFramework/blob/next/packages/runtime/container-runtime/src/containerRuntime.ts#L386) as part of `ContainerRuntimeOptions`.

Reply:
@justus-camp - Did you already have a plan/ETA to rebase op compression to main, or were you saying that this PR is a motivation for moving op compression to main?